src/pages/products-page/HeaderHome.jsx

```diff
@@ -22,24 +23,25 @@ export default function HeaderHome() {
         </div>
         <div className="header-icons">
           <div className="ha-nav-icons">
-            <a href="/login_page/login_page.html">
+            <Link to="/login">
```

Comment:
You have used link in multiple places . Using useNavigate would be a better idea.

Reply:
okay will check in next PR during adding functionality